float64 base_brazo1
float64 base_brazo2
float64 base_brazo3
float64 codo1_a
float64 codo1_b
float64 codo2_a
float64 codo2_b
float64 codo3_a
float64 codo3_b
float64 act_x
float64 act_y
float64 act_z